boot/freeldr/freeldr/arch/i386/pcmem.c

```diff
@@ -254,6 +264,10 @@ PcMemGetBiosMemoryMap(PFREELDR_MEMORY_DESCRIPTOR MemoryMap, ULONG MaxMemoryMapSi
 
     while (PcBiosMapCount < MAX_BIOS_DESCRIPTORS)
     {
+        /* ACPI 3.0/4.0: Set Extended Attributes to enabled/valid by default, in case entry has no E.A.. */
+        ((PBIOS_MEMORY_MAP)BIOSCALLBUFFER)->ExtendedAttributesAsULONG = 0;
```

Comment:
How about RtlZeroMemory'ing the whole structure?

Reply:
I could if you want, but for what actual purpose? Extra safety for very broken BIOS?